Support/cmp/Modules/FindEigen.cmake

```diff
@@ -16,6 +16,7 @@ endif()
 
 SET(EIGEN_INCLUDE_SEARCH_DIRS
   ${EIGEN_INSTALL}/include/eigen3
+  /usr/include/eigen3
```

Comment:
Idea was to run cmake without setting all this variables, because on Linux all could be found under /usr when the packages are installed.

Another idea would be to define a path which should be searched if the lib-variables are not set, or to tell cmake it should use /usr/, for instance:

cmake -Dsearch_usr=1 ../DREAM3D